output/missing-source-info.ts

```diff
@@ -42,6 +42,15 @@ const missingSources: { [key: string]: number[] } = {
   blackarmory: [
     2533990645, // Blast Furnace
   ],
+  brave: [
+    211732170, // Hammerhead
+    243425374, // Falling Guillotine
+    570866107, // Succession
+    2228325504, // Edge Transit
+    2499720827, // Midnight Coup
+    3757612024, // Luna's Howl
+    3851176026, // Elsie's Rifle
```

Comment:
what you're seeing "pulled in" here, are things d2ai found, that _don't_ have a source hash, but are probably the same item as ones that _do_
so it marks these specific item hashes, as ones which are `brave` but can't be categorized using a source hash